### PR TITLE
Typos from Perl5 (2001-2007)

### PR DESCRIPTION
**Pull request by Fourchaux**
Some typos from https://perl5.git.perl.org/perl.git/search/HEAD?s=typo;st=free

### CODE REVIEW

**Comment by Fourchaux** [comment]
For @peternewman (maybe?)

The PRs for "Typos Perl5 (2008-2019)" are almost finished.

The year 2011 is a "monstrosity" (350+ typos). Ouch...

Question: 
Can I send a single PR for this year (my preference !) or do I have to split into several PRs?

**Comment by peternewman** [comment]
> The PRs for "Typos Perl5 (2008-2019)" are almost finished.
> 
> The year 2011 is a "monstrosity" (350+ typos). Ouch...
> 
> Question:
> Can I send a single PR for this year (my preference !) or do I have to split into several PRs?

It's sort of a case of manageability @Fourchaux . After maybe 50-100 items, it can get weary to review them all. I assume you're currently planning to do 2008-2010, 2011, 2012-2019? If you look at the way @Gelma split some up previously:
https://github.com/codespell-project/codespell/pulls?q=is%3Apr+is%3Aclosed+author%3Agelma

I think basically you can slice it up how you want, but you'll get more effective reviews if it's more bitesize, so someone can do it in a free 10-20 mins, rather than having to stop part way through.

Are you doing it manually, or have you got a script that's generating it? You could always make one big diff and then slice it up from there.

**Comment by Fourchaux** [comment]
@peternewman

**Comment by Fourchaux** [comment]
@peternewman 

OK. So, I will make PRs with 70-100 items.

Thank you.

I am doing it manually. No (Perl !) script is used ...

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2102,6 +2124,7 @@ canadan->canadian
 canbe->can be
 cancelability->cancellability
 cancelaltion->cancellation
+cancelled->canceled
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: FYI @Fourchaux , this is valid one side of the pond:
https://www.grammarly.com/blog/canceled-vs-cancelled/

Removed in https://github.com/codespell-project/codespell/pull/877 .

**Pull request merged**